wsmaster/integration-tests/postgresql-tck/pom.xml

```diff
@@ -49,11 +49,6 @@
             <groupId>org.eclipse.che.infrastructure</groupId>
             <artifactId>infrastructure-kubernetes</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.eclipse.che.multiuser</groupId>
-            <artifactId>che-multiuser-machine-authentication</artifactId>
```

Comment:
Should not be the following lines be removed as well?
https://github.com/eclipse/che/blob/c6f0751c740bb4397aa04d9a62b4000596c8794a/wsmaster/integration-tests/postgresql-tck/pom.xml#L167-L171